js_modules/dagit/packages/core/src/plugins/ipynb.tsx

```diff
@@ -8,5 +9,13 @@ export const SidebarComponent: React.FC<IPluginSidebarProps> = (props) => {
   const notebookPath = metadata.find((m) => m.key === 'notebook_path');
   const repoLocName = props.repoAddress?.location;
 
-  return <NotebookButton path={notebookPath?.value} repoLocation={repoLocName || ''} />;
+  return (
+    <Box padding={{horizontal: 16}}>
```

Comment:
can i add the same padding at the bottom?
<img width="282" alt="image" src="https://user-images.githubusercontent.com/4531914/200971702-4611f603-0f90-465c-86c5-84d28877618e.png">

Reply:
done!